jp2/src/lib.rs

```diff
@@ -1742,15 +2274,50 @@ impl fmt::Display for EnumeratedColourSpaces {
             f,
             "{}",
             match self {
+                EnumeratedColourSpaces::BiLevel => "Bi-level",
+                EnumeratedColourSpaces::YCbCr1 => "YCbCr(1)",
+                EnumeratedColourSpaces::YCbCr2 => "YCbCr(2)",
+                EnumeratedColourSpaces::YCbCr3 => "YCbCr(3)",
+                EnumeratedColourSpaces::PhotoYCC => "PhotoYCC",
+                EnumeratedColourSpaces::CMY => "CMY",
+                EnumeratedColourSpaces::CMYK => "CMYK",
+                EnumeratedColourSpaces::YCCK => "YCCK",
+                EnumeratedColourSpaces::CIELab {
```

Comment:
Nit: Use ```..``` to ignore, https://doc.rust-lang.org/book/ch19-03-pattern-syntax.html#remaining-parts-of-a-value-with- 

ie replace:

```
                EnumeratedColourSpaces::CIELab {
                    rl: _,
                    ol: _,
                    ra: _,
                    oa: _,
                    rb: _,
                    ob: _,
                    il: _,
                } => "CIELab",
```
with
```
                EnumeratedColourSpaces::CIELab { .. } => "CIELab",
```

Reply:
Fixed (and in a couple of other places).